pkg/util/kubernetes/apiserver/util.go

```diff
@@ -42,7 +52,7 @@ func SyncInformers(informers map[InformerName]cache.SharedInformer, extraWait ti
 				end := time.Now()
 				cacheSyncTimeouts.Inc()
 				log.Warnf("couldn't sync informer %s in %v (kube_cache_sync_timeout_seconds: %v)", name, end.Sub(start), timeoutConfig)
-				return fmt.Errorf("couldn't sync informer %s in %v", name, end.Sub(start))
```

Comment:
any reason to drop the time in the error, because it's in the log above?

Reply:
Yes, it's to allow us to handle the error more easily here:
https://github.com/DataDog/datadog-agent/pull/28512/files#diff-7a585acdf88c85cf7bf7b21828503aa376757b0927f5cd6394f8b44bbd65bebeR474-R481

Reply:
Yes, but I actually want to only catch `SyncInformersError` that are specific to the `ValidatingWebhooksInformer` while at the same time not having to define another error type like `SyncValidatingInformersError`, as this would also complexify the code of the `SyncInformers` method.